src/components/FormikMultiSelect.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { InputLabel, Select, MenuItem } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+const MenuProps = {
+  getContentAnchorEl: null, //TODO: remove this when material-ui gets updated to version 5
+  anchorOrigin: {
+    vertical: 'bottom',
+    horizontal: 'left',
+  },
+  PaperProps: {
+    style: {
+      maxHeight: 250,
+      width: 250,
+    },
+  },
+};
+
+function FormikMultiSelect(props) {
+  const { label, id, labelStyle, formik, options } = props;
+
+  return (
+    <>
+      <InputLabel className={labelStyle}>{label}</InputLabel>
+      <Select
+        multiple
+        fullWidth
+        id={id}
+        variant="outlined"
+        value={formik.values.requirements[id]}
+        onChange={(e) =>
+          formik.setFieldValue(
+            `requirements.${id}`,
+            e.target.value,
+            /* shouldValidate = */ false
+          )
+        }
+        MenuProps={MenuProps}>
+        {options.map((name) => (
+          <MenuItem key={name} value={name}>
+            {name}
```

Comment:
Something to consider here is that the value may be different from the display name. E.g. "High School Senior" may map to the value `GradeLevel.HS_SENIOR` which under the hood may be the value `12`. I suggest thinking about how to support this use case. You might want to make `options` a dictionary object and do something like:
```suggestion
        {options.map(([name, value]) => (
          <MenuItem key={value} value={value}>
            {name}
```